Resources/Prototypes/_Harmony/Maps/barratry.yml

```diff
@@ -9,12 +9,14 @@
       stationProto: StandardNanotrasenStation
       components:
         - type: StationNameSetup
-          mapNameTemplate: '{0} Barratry {1}'
+          mapNameTemplate: '{0} Pioneer {1}'
```

Comment:
Why rename it?

Reply:
I have plans to develop a "clean" version of the station in future and Pioneer will be the name of the station for both maps. Just a precursor to that eventual development. I can undo if needed.

Reply:
No need. Just checking in case of bad copy paste